modules/nf-core/htsnimtools/vcfcheck/tests/nextflow.config

```diff
@@ -0,0 +1,3 @@
+process {
+    ext.args = "-c 100"
```

Comment:
Why are you setting this in the test? (haven't looked what it does)

Reply:
Ah I forgot to remove it :p  I was seeing if that would make better output but it didn't change anything. I'll remove it!